chart/templates/mayastor/apis/api-rest-deployment.yaml

```diff
@@ -53,6 +53,9 @@ spec:
             - "--core-grpc=https://{{ .Release.Name }}-agent-core:50051"
             - "--ansi-colors={{ .Values.base.logging.color }}"
             - "--fmt-style={{ include "logFormat" . }}"
+            {{- if .Values.apis.rest.healthProbes.readiness.enabled }}
+            - "--core-health-freq={{ .Values.apis.rest.healthProbes.readiness.agentCoreProbeFreq }}"
```

Comment:
If it's not specified does it disable the core health freq or does it just not change the freq?

Reply:
Doesn't disable it, it just uses the default frequency of 2m on the binary.
However, it also disable readiness probe, so in effect, it will call the core Node endpoint once on new(), and then never again. Updates to the cache only happen on `get_or_update` calls.